### DISCUSSION TITLE
The chat list is only halfway through on a vertical display.

### DISCUSSION THREAD

**Discussion by haoblackj** [danny-avila/LibreChat#4026] [General]
Hi,there.

I always use LibreChat on a vertical display, and when I reload or reopen it, the chat list in the left pane is only halfway down (like the attached image).

![image](https://github.com/user-attachments/assets/a3752e9c-11af-42e0-80a6-cd7563cc1af1)

The problem is that even if I scroll down in this state, I can't go back to the list of past chats. So every time I open LibreChat, I have to take the tab to the landscape display and scroll it. Then the list of chats will appear at the bottom, and you can go back to the past chats by scrolling. Once this is done, you will be able to go back to past chats even if you bring the tab back to the portrait display.

I would like to be able to operate in a vertical display in the same way as in a horizontal display.

(I'm not an English speaker, so I use automatic translation. Sorry if it's hard to read)

**Comment by danny-avila** [comment]
Thanks, I understand the issue, hope to mitigate this

**Reply by haoblackj** [reply]
Sorry if I am rushing you.
How about a correction on this topic then?
If there is any additional information that should be presented, I will present it.

I am not an English speaker and use an automatic translation tool. Sorry if my English is not good enough.

**Comment by mikeS141618** [comment]
Ah, thank you for providing an workaround of just putting the window in landscape! 
I've been trying to figure out the words to search for a chat about a month back, but with this work around I can scroll to it! 

Having the same issue in portrait, I can't scroll to an old chat that's not listed in the first 25 chats no matter if It's today/week/month.
